src/components/Form.js

```diff
@@ -135,8 +135,34 @@ function Form(props) {
 
             // Validate the input for html tags. It should supercede any other error
             _.each(trimmedStringValues, (inputValue, inputID) => {
+
+                const m = inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+                const matches = inputValue.match(CONST.VALIDATE_FOR_HTML_TAG_REGEX);
+
+                // add any html tags that are considered valid workspace names to ignoreRegxList
+                const ignoreRegxList = CONST.VALID_WORKSPACENAMES;
+                let isMatch = ignoreRegxList.some(r => r.test(inputValue));
+                const leadingSpaceRegex = inputValue.search(CONST.VALIDATE_FOR_LEADINGSPACES_HTML_TAG_REGEX);
+
                 // Return early if there is no value OR the value is not a string OR there are no HTML characters
-                if (!inputValue || !_.isString(inputValue) || inputValue.search(CONST.VALIDATE_FOR_HTML_TAG_REGEX) === -1) {
+                if (!inputValue || !_.isString(inputValue) || (leadingSpaceRegex === -1 && m === -1) ) {
+                    return;
+                }
+
+                if(matches)
+                {
+                    for(let x of matches)
+                    {
+                        isMatch = ignoreRegxList.some(r => r.test(x));
+                        if(isMatch == false)
+                        {
+                            break;
+                        }
+                    }
+                }
```

Comment:
I assume you are using standard for-look in order to use the `break` command?

Reply:
you got it, I added comments to better explain the code. basically the original regex does not catch leading spaces so if there is any invalid input along with valid input, it needs to break.